src/Components/Blazor/Build/src/Core/RuntimeDependenciesResolver.cs

```diff
@@ -77,28 +84,23 @@ public class AssemblyResolutionContext
             internal void ResolveAssemblies()
             {
                 var visitedAssemblies = new HashSet<string>();
-                var pendingAssemblies = new Stack<AssemblyNameReference>();
-                pendingAssemblies.Push(Assembly.Definition.Name);
+                var pendingAssemblies = new Stack<string>();
```

Comment:
Are these fully-qualified assemblies names (with versions-n-stuff? or just the good part?)

Reply:
Just the names. I think we have a comment somewhere in here that says we don't support multiple versions.